### PR TITLE
Add financials panel & Add AH treasury withdrawal

### PR DESCRIPTION
**Pull request by echohtp**
**after**
image

### CODE REVIEW

File: src/pages/admin/financials/edit.tsx [hunk 1]
```diff
@@ -0,0 +1,326 @@
...
+}
+
+const AdminEditCreators = ({ marketplace }: AdminEditCreatorsProps) => {
+  const wallet = useWallet()
+  const { connection } = useConnection()
+  const { publicKey, signTransaction } = wallet
+
+  const login = useLogin()
+
+  const [showAdd, setShowAdd] = useState(false)
```
Comment (kespinola) [inline src/pages/admin/financials/edit.tsx:114]: not need yeah?

File: src/pages/admin/financials/edit.tsx [hunk 2]
```diff
@@ -0,0 +1,326 @@
...
+  subdomain: string
+  name: string
+  description: string
+  transactionFee: number
+  creators: { address: string }[]
+  feeWithdrawalDestination: string
+  creator: string
+}
+
+const AdminEditCreators = ({ marketplace }: AdminEditCreatorsProps) => {
```
Comment (kespinola) [inline src/pages/admin/financials/edit.tsx:107]: Change component name.

File: src/pages/admin/financials/edit.tsx [hunk 3]
```diff
@@ -0,0 +1,326 @@
...
+      description: marketplace.description,
+      creators: marketplace.creators.map(({ creatorAddress }) => ({
+        address: creatorAddress,
+      })),
+      feeWithdrawalDestination:
+        marketplace.auctionHouse.feeWithdrawalDestination,
+      transactionFee: marketplace.auctionHouse.sellerFeeBasisPoints,
+      creator: '',
+    },
+  })
```
Comment (kespinola) [inline src/pages/admin/financials/edit.tsx:137]: Not editing anything so don't edit market form object.

**Review by kespinola — COMMENTED** [review]
looking good.

File: src/pages/admin/financials/edit.tsx (hunk 1 above)
Comment (echohtp) [reply src/pages/admin/financials/edit.tsx:114]: correct, not needed, will remove

Comment (echohtp) [reply src/pages/admin/financials/edit.tsx]: correct, not needed, will remove

**Review by kespinola — APPROVED** [review]

**Comment by kespinola** [comment]
In a follow up we can make rpc call to read the current amount in the account and display that to the user.

Also need to show that 50% going to hola and other going to the user @damianr

**Comment by damianr** [comment]
@kespinola here's a link to the designs for this. We can scale back if needed but if we're gonna show which wallets are getting funds we might as well use this layout. https://www.figma.com/file/eACyDSbpcWYoE9ypOZNlf8/Marketplaces?node-id=1307%3A27416

image

**Pull request merged**